ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -204,28 +225,46 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test-hypershift
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift-guest
-- as: aws-ipi-shared-vpc-phz-freeipa-f28
-  cron: 40 1 14 * *
+- as: aws-ipi-private-shared-vpc-phz-sts-f28
+  cron: 32 19 16 * *
   steps:
     cluster_profile: aws-qe
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
+      EXTRACT_MANIFEST_INCLUDED: "true"
     test:
-    - ref: idp-freeipa
     - chain: openshift-upgrade-qe-test
-    workflow: cucushift-installer-rehearse-aws-ipi-shared-vpc-phz
-- as: aws-ipi-shared-vpc-phz-sts-fips-openldap-f28
-  cron: 53 0 6 * *
+    workflow: cucushift-installer-rehearse-aws-ipi-private-shared-vpc-phz-sts
+- as: aws-ipi-proxy-sdn-fips-f28
```

Comment:
If I remember right, we decide to cover `sdn` in chain upgrade, if we also going to cover it in y-stream upgrade, need @zhaozhanqi to confirm.

Reply:
And I also saw `aws-ipi-proxy-sdn-fips-f28` in openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.16.yaml, they are duplicted.

Reply:
@JianLi-RH checked https://github.com/openshift/release/pull/53470, sdn only can be covered in chain upgrade. So I think we need to drop it here.

Reply:
Sure, thanks